### PR TITLE
1468173 - Add bootstrap_on_startup feature

### PR DESCRIPTION
**Pull request by eriknelson**

### CODE REVIEW

**Comment by eriknelson** [comment]
# Test

* Configure catasb  to deploy image with this patch
* Remove (comment out) bootstrap task from catasb setup
* Confirm webUI displays APBs or curl <asb-route>/v2/catalog and confirm contents
* Should additionally see new logs in the asb pod saying broker configured for bootstrap_on_startup

File: etc/example-config.yaml [hunk 1]
```diff
@@ -18,6 +18,7 @@ openshift:
   user: OPENSHIFT_USER
   pass: OPENSHIFT_PASS
 broker:
+  bootstrap_on_startup: true
```
Comment (jmrodri) [inline etc/example-config.yaml]: Good default

**Comment by shawn-hurley** [comment]
Visual ACK

May want to add the configuration to the run_local.sh to auto bootstrap?

File: pkg/app/app.go [hunk 2]
```diff
@@ -147,6 +147,17 @@ func (a *App) Start() {
 		a.Recover()
 	}
 
+	if a.config.Broker.BootstrapOnStartup {
+		a.log.Info("Broker configured to bootstrap on startup")
+		a.log.Info("Attempting bootstrap...")
```
Comment (jmrodri) [inline pkg/app/app.go]: I would've probably combined the above two lines, but just a nit.

File: pkg/broker/broker.go [hunk 3]
```diff
@@ -40,10 +40,11 @@ type Broker interface {
 }
 
 type BrokerConfig struct {
-	DevBroker       bool `yaml:"dev_broker"`
-	LaunchApbOnBind bool `yaml:"launch_apb_on_bind"`
-	Recovery        bool `yaml:"recovery"`
-	OutputRequest   bool `yaml:"output_request"`
+	DevBroker          bool `yaml:"dev_broker"`
+	LaunchApbOnBind    bool `yaml:"launch_apb_on_bind"`
+	BootstrapOnStartup bool `yaml:"bootstrap_on_startup"`
```
Comment (jmrodri) [inline pkg/broker/broker.go]: 👍 yaml matches the actual config value set above.

File: templates/deploy-ansible-service-broker.template.yaml [hunk 4]
```diff
@@ -200,6 +200,7 @@ objects:
       openshift: {}
       broker:
         dev_broker: ${DEV_BROKER}
+        bootstrap_on_startup: ${BOOTSTRAP_ON_STARTUP}
```
Comment (jmrodri) [inline templates/deploy-ansible-service-broker.template.yaml]: good job remembering to update template.

**Comment by eriknelson** [comment]
@shawn-hurley will update that in the master PR, TY for the reminder

**Pull request merged**